src/test/java/it/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMDescriptorIT.java

```diff
@@ -215,7 +215,7 @@ public void testFindRepo() throws Exception {
         BitbucketNamedLink httpCloneUrl = repo.getCloneUrls().stream().filter(url -> "http".equals(url.getName()))
                 .findAny()
                 .orElseThrow(() -> new AssertionError("There should be an http clone url"));
-        assertThat(httpCloneUrl.getHref(), equalTo("http://admin@localhost:7990/bitbucket/scm/project_1/rep_1.git"));
+        assertThat(httpCloneUrl.getHref().replace("admin@", ""), equalTo("http://admin@localhost:7990/bitbucket/scm/project_1/rep_1.git"));
```

Comment:
This doesn't make sense to me. Should the `equalTo` value not have the `admin@`?

Reply:
You are right. I think it was an oversight from Martin.

Reply:
yes my bad, thanks for catching